### DISCUSSION TITLE
Original Xoreos Games

### DISCUSSION THREAD

**Discussion by springheeledjack66** [xoreos/xoreos#584] [Q&A] [answered]
Hi, I was curious about the fitness of Xoreos as an engine for creating original games in the style of old Bioware games, is it possible?

**Comment by DrMcCoy** [comment, marked as answer]
Frankly, that's not something that's on the roadmap, no. Maybe that can be re-evaluation far in the future when xoreos does what it's meant to do, but that doesn't look to be any day soon.
